docs/src/documentation/03-components/05-input/inputfield/03-accessibility.mdx

```diff
@@ -39,10 +98,10 @@ In this example, the error message is automatically associated with the input fi
 If you provide both an `ariaDescribedby` prop and use `error` or `help`, the component automatically combines them, ensuring that screen readers announce all descriptive content:
 
 ```jsx
-  <p id="email-hint" style={{ display: "none", visibility: "hidden" }}>
-    We'll never share your email with anyone else.
-  </p>
-  <InputField label="Email" error="Invalid email format" ariaDescribedby="email-hint" />
+<p id="email-hint" style={{ display: "none", visibility: "hidden" }}>
```

Comment:
In the example above, the `<p>` is put after `InputField` component. I would align the order of elements in these two examples.